File: POO PHP/Aula12/Animal.php
<?php

abstract class Animal {

    //Atributos da Classe
    protected $peso;
    protected $idade;
    protected $membro;
    
    //Metodos Acessores e Modificadores
    public function getPeso() {
        return $this->peso;
    }
    public function getIdade() {
        return $this->idade;
    }
    public function getMembro() {
        return $this->membro;
    }
    public function setPeso($peso): void {
        $this->peso = $peso;
    }
    public function setIdade($idade): void {
        $this->idade = $idade;
    }
    public function setMembro($membro): void {
        $this->membro = $membro;
    }
    
    //Metodos da Classe
    abstract function locomover();
    abstract function alimentar();
    abstract function emitirSom();
}

File: POO JAVA/Aula13/src/aula13/Animal.java
package aula13;

abstract class Animal {

    //Atributos da Classe
    protected float peso;
    protected int idade;
    protected int membro;
    
    //Metodo Acessores e Modificadores
    public float getPeso() {
        return peso;
    }
    public void setPeso(float peso) {
        this.peso = peso;
    }
    public int getIdade() {
        return idade;
    }
    public void setIdade(int idade) {
        this.idade = idade;
    }
    public int getMembro() {
        return membro;
    }
    public void setMembro(int membro) {
        this.membro = membro;
    }
    
    //Metodo da Classe
    public abstract void emitirSom();
}
